config/redirects.yml

```diff
@@ -410,3 +410,4 @@ redirects:
   "/why-teach(*reason)": "/blog"
   "/content/stories(*story)": "/blog"
   "/my-story-into-teaching(*story)": "/blog"
+  "/event-categories/(*category)": "/events"
```

Comment:
@peteryates As per my comments on the Events deployment PR, this doesn't work - we have a TTT page that we want people to go to! Plus I think the others should probably go to pre-filtered lists. Happy to discuss

Reply:
@peteryates This is covered by https://github.com/DFE-Digital/get-into-teaching-app/pull/2613 now so I think needs removing here?